tests/__snapshots__/files.test.js.snap

```diff
@@ -215,12 +215,12 @@ export interface FetchCallbackResponseArray<T, V> {
   resource: Resource<T>;
   /**
    * @deprecated Resolve clear with condition in your fetch api this function
-   *   will be remove
+   * will be remove
    */
```

Comment:
Don't remove space of multi lines second line.

Reply:
We are added this guide from JavaScript Style Guide.
https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

Reply:
May I ask why this was added in the first place?

It's not like this plugin follows the Google JS style guide closely. The indentation of exactly 2 spaces is somewhat arbitrary as well.

I also what to point out that the indentation is semantically invalid Markdown. Consistently indented lines are used for code blocks in MD. This means that other tools like JSDoc and TS may correctly assume that those are code blocks.

That being said, it does look good. But I don't know if good looks justify us breaking other tools.

Reply:
The Markdown don't have tag description JSDoc is not markdown just part of JSDoc description can be like Markdown.

I used Markdown prettier formatter just few commits ago, reverted that, because has bad result, I don't like mistake again, at least on this time we can't use Markdown